packages/jest-cli/src/cli/args.js

```diff
@@ -97,6 +106,15 @@ const options = {
       'paths before executing the test.',
     type: 'string',
   },
+  findRelatedTests: {
+    alias: 'f',
+    description:
+      'Find related tests for the a list of source files passed ' +
+      'in arguments and run tests. Useful for pre-commit hook ' +
+      'integration when you want to run single tests for the ' +
+      'changed source files.',
```

Comment:
Some typos here.

I suggest:

"Find related tests for a list of source files that were passed in as arguments. Useful for pre-commit hook integration to run the minimal amount of tests necessary."